internal/adapters/entrypoints/rest/handlers/delete_trusted_account.go

```diff
@@ -0,0 +1,34 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/rsksmart/liquidity-provider-server/internal/adapters/entrypoints/rest"
+	lpuc "github.com/rsksmart/liquidity-provider-server/internal/usecases/liquidity_provider"
+	"github.com/rsksmart/liquidity-provider-server/pkg"
+)
+
+// NewDeleteTrustedAccountHandler
+// @Title Delete Trusted Account
+// @Description Deletes a trusted account
+// @Param TrustedAccountAddressRequest body pkg.TrustedAccountAddressRequest true "Address of the trusted account to delete"
+// @Success 204 object
+// @Route /management/trusted-accounts [delete]
+func NewDeleteTrustedAccountHandler(useCase *lpuc.DeleteTrustedAccountUseCase) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		var err error
+		request := &pkg.TrustedAccountAddressRequest{}
+		if err = rest.DecodeRequest(w, req, request); err != nil {
+			return
+		} else if err = rest.ValidateRequest(w, request); err != nil {
+			return
+		}
+		err = useCase.Run(req.Context(), request.Address)
+		if err != nil {
```

Comment:
what about the case where the account doesn't exist? Shouldn't it return 404 instead of 500?